sdks/v2-sdk/src/entities/pair.ts

```diff
@@ -25,17 +26,24 @@ export const computePairAddress = ({
   factoryAddress,
   tokenA,
   tokenB,
+  chainId,
 }: {
   factoryAddress: string
   tokenA: Token
   tokenB: Token
+  chainId?: ChainId
 }): string => {
   const [token0, token1] = tokenA.sortsBefore(tokenB) ? [tokenA, tokenB] : [tokenB, tokenA] // does safety checks
-  return getCreate2Address(
-    factoryAddress,
-    keccak256(['bytes'], [pack(['address', 'address'], [token0.address, token1.address])]),
-    INIT_CODE_HASH
-  )
+  const salt = keccak256(['bytes'], [pack(['address', 'address'], [token0.address, token1.address])])
+  const initCodeHash = INIT_CODE_HASH
```

Comment:
the initcodehash is different for zksync 😢

Reply:
we dont have a v2 one (because its not deployed) but it would be different to the EVM one

Reply:
alright, I guess I will abandon this PR for now. I thought the salt and init code hash would be the same, so that we can preempt to change the v2-sdk.